addons/test_mail/models/test_mail_corner_case_models.py

```diff
@@ -182,6 +182,43 @@ def _mail_get_partner_fields(self, introspect_fields=False):
         return ['customer_id']
 
 
+class MailTestStageField(models.Model):
```

Comment:
Would move to ``test_mail_feature_models.py``, which holds some models linked to specific features (as test_mail_models was becoming big, I began splitting it). You can make a section for rot, under PROPERTIES; we try to divide by feature / section so that file is "clear" to read